app/src/main/java/com/bogdan/codeforceswatcher/features/actions/redux/requests/ActionsRequests.kt

```diff
@@ -69,19 +67,17 @@ class ActionsRequests {
             val uiData: MutableList<CFAction> = mutableListOf()
 
             for (action in actions) {
-                if (action.comment != null) {
-                    users?.find { user -> user.handle == action.comment.commentatorHandle }
+                action.comment?.let { comment ->
+                    users?.find { user -> user.handle == comment.commentatorHandle }
                             ?.let { foundUser ->
-                                action.comment.commentatorAvatar = foundUser.avatar
-                                action.comment.commentatorRank = foundUser.rank
+                                comment.commentatorAvatar = foundUser.avatar
+                                comment.commentatorRank = foundUser.rank
                             }
 
-                    action.comment.text = convertFromHtml(action.comment.text)
-                } else {
-                    if (action.timeInMillisecond != action.blogEntry.creationTimeSeconds &&
-                            action.timeInMillisecond != action.blogEntry.modificationTimeSeconds) {
-                        continue
-                    }
+                    comment.text = convertFromHtml(comment.text)
+                } ?: if (action.timeSeconds != action.blogEntry.creationTimeSeconds &&
```

Comment:
This condition isn't very readable. Wdyt about extracting it to variable or method? For example: `isDuplicatedActions`.